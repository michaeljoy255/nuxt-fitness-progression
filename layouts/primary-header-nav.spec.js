import { mount, createLocalVue } from '@vue/test-utils'
import Vuetify from 'vuetify'
import EmptyStub from '../test/stubs/empty-stub'
import PrimaryHeaderNavLayout from './primary-header-nav'

const localVue = createLocalVue()

let wrapper

beforeEach(() => {
  const vuetify = new Vuetify()
  wrapper = mount(PrimaryHeaderNavLayout, {
    stubs: {
      PrimaryHeaderBar: EmptyStub,
      NavDrawer: EmptyStub,
    },
    localVue,
    vuetify,
  })
})

afterEach(() => {
  wrapper.destroy()
})

describe('Layout > header-nav', () => {
  test('matches the snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
