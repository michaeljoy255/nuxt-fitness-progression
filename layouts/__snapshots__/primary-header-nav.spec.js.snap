// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout > header-nav matches the snapshot 1`] = `
"<div data-app=\\"true\\" class=\\"v-application v-application--is-ltr theme--light\\" id=\\"app\\">
  <div class=\\"v-application--wrap\\">
    <div></div>
    <div></div>
    <main class=\\"v-main\\" style=\\"padding: 0px 0px 0px 0px;\\">
      <div class=\\"v-main__wrap\\">
        <div></div>
      </div>
    </main>
  </div>
</div>"
`;
